marc_to_solr/spec/fixtures/sample13.mrx

```diff
@@ -152,7 +152,7 @@
       <marc:subfield code="a">Collection Archives (Institut national de la communication audiovisuelle (France))</marc:subfield>
     </marc:datafield>
     <marc:datafield tag="852" ind1="8" ind2=" ">
-      <marc:subfield code="b">rcpqk</marc:subfield>
+      <marc:subfield code="b">recap$pq</marc:subfield>
       <marc:subfield code="h">LS- 11479</marc:subfield>
```

Comment:
Did we decide to go with the `$` or the `-` between location and code? oh ok I see a follow up issue on that https://github.com/pulibrary/marc_liberation/issues/1012. @eliotjordan is it better to update it now with the `-` and regenerate the tables? @hackartisan what do you think?

Reply:
The codes will have the `$`. The issue is to add the `-` into the labels. I don't think it's important to do it before merging this based on discussions I've had with @eliotjordan. I do think it will be good to do it on the soon side.